src/main/scala/Garage/Garage.scala

```diff
@@ -1,71 +1,83 @@
 package Garage
+
 import Garage.DBConnection._
-import org.mongodb.scala.bson.collection.immutable.Document
-import scala.collection.mutable.ListBuffer
+import Garage.DBMethods._
+import org.mongodb.scala._
+import org.bson.BsonDocument
 
 class Garage() {
-  var isOpen = true
-  var vehicleList = new ListBuffer[Vehicle]()
-  var registeredEmployeesList = new ListBuffer[Person]()
 
-  def viewGarageContents = {
-    println("Vehicles in garage: " + vehicleList)
-    println("Employees in garage: " + registeredEmployeesList)
+  def openConnection(collectionName:String):MongoCollection[Document] = {
+    val client = getClient("mongodb://localhost")
+    val db = getConnectionDatabase(client,"garage")
+    val collection = getConnectionCollection(db, collectionName)
+      collection
+  }
+
+  def closeConnection():Unit = {
+    val client = getClient("mongodb://localhost")
```

Comment:
`"mongodb://localhost"` you're using this a few times here, have a constants file from which you could get it rather than duplicate it